code/modules/mining/machine_vending.dm

```diff
@@ -203,33 +203,36 @@
 	return ..()
 
 /obj/machinery/vendor/mining/proc/RedeemVoucher(obj/item/mining_voucher/voucher, mob/redeemer)
-	var/items = list("Survival Capsule and Explorer's Webbing", "Resonator Kit", "Minebot Kit", "Extraction and Rescue Kit", "Crusher Kit", "Mining Conscription Kit")
+	var/items = list("Asteroid Miner Kit", "Crusher Kit", "Survivalist Kit", "Minebot Kit", "Mining Conscription Kit")
 
 	var/selection = input(redeemer, "Pick your equipment", "Mining Voucher Redemption") as null|anything in sortList(items)
 	if(!selection || !Adjacent(redeemer) || QDELETED(voucher) || voucher.loc != redeemer)
 		return
 	var/drop_location = drop_location()
 	switch(selection)
-		if("Survival Capsule and Explorer's Webbing")
-			new /obj/item/storage/belt/mining/vendor(drop_location)
-		if("Resonator Kit")
-			new /obj/item/storage/firstaid/brute(drop_location)
-			new /obj/item/resonator(drop_location)
-		if("Minebot Kit")
-			new /mob/living/simple_animal/hostile/mining_drone(drop_location)
-			new /obj/item/weldingtool/hugetank(drop_location)
-			new /obj/item/clothing/head/welding(drop_location)
-			new /obj/item/borg/upgrade/modkit/minebot_passthrough(drop_location)
-		if("Extraction and Rescue Kit")
+		if("Asteroid Miner Kit")
+			new /obj/item/clothing/suit/space/hardsuit/mining(drop_location)
+			new /obj/item/tank/internals/oxygen(drop_location)
+			new /obj/item/gps/mining(drop_location)
 			new /obj/item/extraction_pack(drop_location)
 			new /obj/item/fulton_core(drop_location)
-			new /obj/item/stack/marker_beacon/thirty(drop_location)
 		if("Crusher Kit")
 			new /obj/item/extinguisher/mini(drop_location)
 			new /obj/item/kinetic_crusher(drop_location)
+		if("Survivalist Kit")
+			new /obj/item/survivalcapsule/encampment(drop_location)
+			new /obj/item/wormhole_jaunter(drop_location)
+			new /obj/item/hivelordstabilizer(drop_location)
+		if("Minebot Kit")
+			new /mob/living/simple_animal/hostile/mining_drone(drop_location)
+			new /obj/item/weldingtool/hugetank(drop_location)
+			new /obj/item/clothing/head/welding(drop_location)
+			new /obj/item/borg/upgrade/modkit/minebot_passthrough(drop_location)
+			new /obj/item/slimepotion/slime/sentience/mining(drop_location)
 		if("Mining Conscription Kit")
 			new /obj/item/storage/backpack/duffelbag/mining_conscript(drop_location)
```

Comment:
I think the kits are too powerful.
Asteroid miner kit should be just the suit and internals imo.
Survivalist should have a smaller capsule.
Minebot should lose the sentience.

Reply:
* Downgraded the survalist kit to a survival capsule per request
* Asteroid mining absolutely requires fultons in order to not risk being permanently stranded. There's no radio if the shuttle gets recalled. Removed GPS purely because it doesn't actually do anything for an asteroid miner though. 
* Non-sentient minebot is incapable of doing anything, so it's a waste of a voucher that's supposed to help get you started. It's potentially a waste even as-is just because there's no guarantee anyone will take the bot. If sentience is a breaking point, the kit is better off being removed.

Reply:
If a bacon's PR for supercruise v2 change is merged, they'll need to use their GPS letting people know they're stranded. I don't feel preventing permanently stranded should be a thing. that's all their fault, and should do better next time.

Reply:
If you're taking the shuttle to mine asteroids and someone on station recalls the shuttle knowing it is no longer being used to travel to lavaland, I kinda don't see that as the fault of the asteroid miner. 

That said, after discord discussion the Fultons were replaced with a single-use jaunter instead.